targets/csharp/templates/PlayFabSDK.csproj.ejs

```diff
@@ -54,6 +54,7 @@
     <Compile Include="source\PluginContract.cs" />
     <Compile Include="source\PluginManager.cs" />
     <Compile Include="source\Uunit\PlayFabApiTest.cs" />
+    <Compile Include="source\Uunit\PluginManagerTest.cs" />
```

Comment:
>PluginManagerTest [](start = 35, length = 17)

I know we talked about adding this to UnitTestRunner project instead, but there are complications with that. I can show offline, it will be faster.

Reply:
No worries, the tests do what they're meant to do, and they work. So good enough!  :D